### DISCUSSION TITLE
馬斯克暫停收購twitter

### DISCUSSION THREAD

**Discussion by yuhao112** [1101-GS4719-C/Python-Community-GS4719-C#598] [科技新知]
https://www.inside.com.tw/article/27673-twitter-deal-pending
https://www.inside.com.tw/article/27480-jack-twitter-and-elon-musk
前陣子馬斯克才說要收購twitter，其中也開放了很多被限制的帳戶，在twitter的言論自由也變得更加受保障，twitter創辦人也公開表明支持他這樣的決定(讓twitter脫離華爾街的控制)。然而，這當一切利好消息出來後，馬斯克又說明暫緩收購，他說到因為twitter裡面假帳號只佔了5%才讓他下定決心停止，想問一下有人知道這有甚麼關聯嗎

**Comment by JSHT** [comment]
看文中的意思是，twitter 預估說不到 5% 的活躍用戶是假帳號。但是 Mask 說要收購後，twitter 說第一季投放的廣告可觸及到 2.29 億用戶(沒區分真假帳號)。可能是 Mask 那邊認為 twitter 活躍的真用戶沒有那麼多，假帳號的占比可能不只 5%，所以想等更詳細的報告出來。

**Comment by HsuChen88** [comment]
還是其實是公然炒股

**Comment by ghost** [comment]
我的twi寶... 說真的目前暫緩也許是好事 很多事情可能得多開幾次股東會商討

**Comment by handsomeniko** [comment]
他真的很懂炒作

**Comment by bloodtrail0511** [comment]
可能是在忙著處理中國[伊隆馬](https://www.mirrormedia.mg/story/20220516edi011/)吧XD

**Comment by ThomasMeng0515** [comment]
前陣子Musk還說要收購可口可樂，把毒品配方原料放回去，真的很有想法哈哈

**Comment by Sazmafuji** [comment]
twitter人口大部分只有美國和日本在用，所以才先暫停?

**Comment by leon10456** [comment]
炒股成功

**Comment by liangyinglly** [comment]
馬斯克真是個會製造話題的人

**Comment by jason79461385** [comment]
我比較在意依龍罵何時要帶我們騷兩圈

**Comment by bloodtrail0511** [comment]
HI AV灣 艾姆伊隆罵~
